csv-bulk-signature/src/main.ts

```diff
@@ -189,25 +186,22 @@ async function getCSVRows(path: string) {
   });
 }
 
-async function filterCompleteContacts(contacts: Contact[]) {
-  // filter out the first row (placeholder)
-  const filteredContacts = contacts.filter((contact) => {
-    // @ts-ignore - a hack to skip the first row
-    return contact["Brand*"] !== "use dropdown to select a brand";
-  });
-
-  // check if all required fields are present and log out skipped rows
-  const contactsWithRequiredFields = filteredContacts.filter((contact) => {
-    const hasRequiredData = checkRequiredFields(contact);
-    if (!hasRequiredData) {
-      skippedRows.push(contact["Full Name*"]);
-    } else {
-      processedRows.push(contact["Full Name*"]);
-    }
-    return hasRequiredData;
-  });
+function skipFirstPlaceholderRow(contact: Contact) {
+  // @ts-ignore - a hack to skip the first row
+  return contact["Brand*"] !== "use dropdown to select a brand";
+}
 
-  return contactsWithRequiredFields;
+function filterCompleteContacts(
+  contact: Contact,
+  { skippedRows, processedRows }: StatusReport
+) {
+  const hasRequiredData = checkRequiredFields(contact);
+  if (!hasRequiredData) {
+    skippedRows.push(contact["Full Name*"]);
+  } else {
+    processedRows.push(contact["Full Name*"]);
+  }
+  return hasRequiredData;
 }
```

Comment:
this function was doing two things:
- refactor into two seperate filter functions